e2e/commands/start/start_test.go

```diff
@@ -96,9 +99,16 @@ var _ = Describe("running binary Horusec with start parameter", func() {
 		})
 	})
 
+	When("--project-path is passed", func() {
+		It("Checks if the project path property was set", func() {
+			Expect(session.Out.Contents()).To(ContainSubstring(`\"disable_docker\": true`))
```

Comment:
substring should be about `\"project_path\"` and not `\"disable_docker\":`

Reply:
Something wrong is not right 🧐, Fixed !